### PR TITLE
README and other minor fixes

### PR DESCRIPTION
**Pull request by venkat-443**
operator readme: updated with requirement of kubernetes 1.8 or higher
server_quick_setup readme: removed redundant label command and fixed typos
definitions: fixed minor issues faced while deploying.
    API url: missing : 
    Tenant default value changed to "My Tenant"
    Claim and PV size mismatch causing PVC bound failure.

### CODE REVIEW

File: operator/README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Quobyte operator
-Deploys quobyte clients on to the nodes configured in Custom Resuource QuobyteClient.
+Deploys quobyte clients on to the nodes configured in Custom Resuource QuobyteClient.
```
Comment (johscheuer) [inline operator/README.md]: Typo and `Definition` ist missing.

Quobyte should start with an upper case Q?

File: operator/README.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Quobyte operator
-Deploys quobyte clients on to the nodes configured in Custom Resuource QuobyteClient.
+Deploys quobyte clients on to the nodes configured in Custom Resuource QuobyteClient.  
+`Note:` Operator requires Custom Resource Definitions (CRD), which is only available from kubernetes 1.8
```
Comment (johscheuer) [inline operator/README.md]: "which are only"?

Comment (venkat-443) [reply operator/README.md]: Made correction.

File: operator/README.md (hunk 1 above)
Comment (venkat-443) [reply operator/README.md]: Thank you. Corrected it.

File: operator/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Quobyte operator
-Deploys quobyte clients on to the nodes configured in Custom Resuource QuobyteClient.  
-`Note:` Operator requires Custom Resource Definitions (CRD), which is only available from kubernetes 1.8
+Deploys Quobyte clients on to the nodes configured in Custom Resuource QuobyteClient.  
+`Note:` Operator requires [Custom Resource Definitions (CRD)](https://kubernetes.io/docs/concepts/api-extension/custom-resources/), which are available only from kubernetes 1.8
```
Comment (casusbelli) [inline operator/README.md]: '...only for Kubernetes 1.8 and newer.'

**Pull request merged**